sqlite/ext/comdb2/tables.c

```diff
@@ -255,6 +255,8 @@ int comdb2SystblInit(
     rc = sqlite3_create_module(db, "comdb2_timepartshards", &systblTimepartShardsModule, 0);
   if (rc == SQLITE_OK)
     rc = sqlite3_create_module(db, "comdb2_timepartevents", &systblTimepartEventsModule, 0);
+  if (rc == SQLITE_OK)
+    rc = sqlite3_create_module(db, "comdb2_transaction_log", &systblTransactionLogModule, 0);
```

Comment:
A minor suggestion: a general convention is to use plural name for the system tables.